### ISSUE TITLE
Review for WCAG accessibility

### ISSUE THREAD

**Issue by brucecrevensten** [ua-snap/data-api#700]
Goal: meet/exceed https://www.w3.org/TR/WCAG21/
Review tool: https://wave.webaim.org

(Should be an easy one)
